### PR TITLE
job tweaks + xenobio/toxi icons

### PR DESCRIPTION
**Pull request by Jolly-66**
more tweaks because i forgot the initial batch 

also added icons n shit after an hour of us fussing over it

### CODE REVIEW

File: jollystation_modules/code/game/objects/items/card_ids.dm [hunk 1]
```diff
@@ -0,0 +1,2 @@
+/obj/item/card/id/module
+  icon = 'jollystation_modules/icons/obj/card.dmi'
```
Comment (MrMelbert) [inline jollystation_modules/code/game/objects/items/card_ids.dm:2]:
```suggestion
  icon = 'jollystation_modules/icons/obj/card.dmi'
  
```

File: jollystation_modules/code/game/objects/items/devices/radio/headset.dm [hunk 2]
```diff
@@ -4,3 +4,8 @@
 	icon_state = "com_headset"
 	keyslot = new /obj/item/encryptionkey/heads/hosv
 	
+/obj/item/radio/headset/heads/headset_qm
+	name = "supply radio headset"
+	desc = "A headset used by the QM."
+	icon_state = "cargo_headset"
+	keyslot = new /obj/item/encryptionkey/heads/qm
```
Comment (MrMelbert) [inline jollystation_modules/code/game/objects/items/devices/radio/headset.dm:11]:
```suggestion
	keyslot = new /obj/item/encryptionkey/heads/qm
	
```

File: jollystation_modules/code/modules/jobs/job_types/toxicologist.dm [hunk 3]
```diff
@@ -26,3 +26,5 @@
 /datum/outfit/job/scientist/toxicologist
 	name = "Toxicologist"
 	jobtype = /datum/job/toxicologist
+
+	id = /obj/item/card/id/module
```
Comment (MrMelbert) [inline jollystation_modules/code/modules/jobs/job_types/toxicologist.dm:30]:
```suggestion
	id = /obj/item/card/id/module
	
```

File: jollystation_modules/code/modules/jobs/job_types/xenobiologist.dm [hunk 4]
```diff
@@ -25,3 +25,5 @@
 /datum/outfit/job/scientist/xenobiologist
 	name = "Xenobiologist"
 	jobtype = /datum/job/xenobiologist
+	
+	id = /obj/item/card/id/module
```
Comment (MrMelbert) [inline jollystation_modules/code/modules/jobs/job_types/xenobiologist.dm:29]:
```suggestion
	id = /obj/item/card/id/module
	
```

File: jollystation_modules/code/modules/mob.dm [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+    hud_list = list()
+    for(var/hud in hud_possible)
+        var/hint = hud_possible[hud]
+        switch(hint)
+            if(HUD_LIST_LIST)
+                hud_list[hud] = list()
+            else
+                var/image/I = image('jollystation_modules/icons/mob/hud.dmi', src, "")
+                I.appearance_flags = RESET_COLOR|RESET_TRANSFORM
+                hud_list[hud] = I
```
Comment (MrMelbert) [inline jollystation_modules/code/modules/mob.dm:11]:
```suggestion
                hud_list[hud] = I
                
```

File: jollystation_modules/code/modules/mob/living/living.dm [hunk 6]
```diff
@@ -0,0 +1,3 @@
+/mob/living/prepare_huds()
+    . = ..()
+    prepare_jollystation_huds()
```
Comment (MrMelbert) [inline jollystation_modules/code/modules/mob/living/living.dm:3]:
```suggestion
    prepare_jollystation_huds()
    
```

**Review by MrMelbert — CHANGES_REQUESTED** [review]

File: jollystation_modules/code/modules/mob/living/living.dm (hunk 6 above)
Comment (MrMelbert) [reply jollystation_modules/code/modules/mob/living/living.dm:3]: pretty sure this application can be refactored to be less jank but that's a problem for future me

File: code/modules/jobs/access.dm [hunk 7]
```diff
@@ -372,7 +372,7 @@
 	return list("Assistant", "Captain", "Head of Personnel", "Bartender", "Cook", "Botanist", "Quartermaster", "Cargo Technician",
 				"Shaft Miner", "Clown", "Mime", "Janitor", "Curator", "Lawyer", "Chaplain", "Chief Engineer", "Station Engineer",
 				"Atmospheric Technician", "Chief Medical Officer", "Medical Doctor", "Paramedic", "Chemist", "Geneticist", "Virologist", "Psychologist",
-				"Research Director", "Scientist", "Roboticist", "Head of Security", "Warden", "Detective", "Security Officer", "Prisoner")
+				"Research Director", "Scientist", "Xenobiologist", "Toxicologist", "Roboticist", "Head of Security", "Warden", "Detective", "Security Officer", "Prisoner") //NON-MODULAR CHANGE
```
Comment (MrMelbert) [inline code/modules/jobs/access.dm:375]: i think in the future additions to this should be done through it's own proc but probably not now

Comment (Jolly-66) [reply code/modules/jobs/access.dm:375]: noting for the future

**Review by MrMelbert — APPROVED** [review]

**Pull request merged**